Packs/CrowdStrikeFalcon/Playbooks/playbook-Command_and_Scripting_Interpreter.yml

```diff
@@ -0,0 +1,1717 @@
+      scriptName: CommandLineAnalysis
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "101"
+    scriptarguments:
+      command_line:
+        simple: ${inputs.CommandLines}
+    reputationcalc: 2
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1410,
+          "y": -360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "101":
+    id: "101"
+    taskid: 8317cebf-6485-4687-8ef6-9f43ac3cb36b
+    type: regular
+    task:
+      id: 8317cebf-6485-4687-8ef6-9f43ac3cb36b
+      version: -1
+      name: Enrich Indicators Extracted From CommandLine
+      description: commands.local.cmd.enrich.indicators
+      script: Builtin|||enrichIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "39"
+    scriptarguments:
+      indicatorsValues:
+        complex:
+          root: CommandLineAnalysis.analysis.original
+          accessor: indicators
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: CommandLineAnalysis.analysis.decoded.indicators
+                iscontext: true
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1410,
+          "y": -190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "102":
+    id: "102"
+    taskid: 45585eb8-b5bf-4056-8efa-158221f79f0e
+    type: regular
+    task:
+      id: 45585eb8-b5bf-4056-8efa-158221f79f0e
+      version: -1
+      name: Create new incident for investigating the indicators
+      description: commands.local.cmd.create.inc
+      script: Builtin|||createNewIncident
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "106"
+    scriptarguments:
+      description:
+        simple: "We've identified the following hostnames being associated with malicious indicators. \nThe list of the endpoints:\n${Endpoint.ID}\n\nThe list of malicious indicators associated with those endpoint from above:\n${Indicators(val.Score>2).Value}\n\nPlease check the investigation id: ${incident.id}"
+      details:
+        simple: "We've identified the following hostnames being associated with malicious indicators. \nThe list of the endpoints:\n${Endpoint.ID}\n\nThe list of malicious indicators associated with those endpoint from above:\n${Indicators(val.Score>2).Value}\n\nPlease check the investigation id: ${incident.id}"
```

Comment:
Instead of using the endpoint IDs lets use the endpoint hostnames

Reply:
you can use the field you used to set the names in the task before "affectedhosts"

Reply:
Great idea, changing